src/components/20-molecules/accordion/index.js

```diff
@@ -40,6 +42,8 @@ class AXAAccordion extends LitElement {
     if (!this.disabled) {
       this.open = !this.open;
       this.toggleAnimation();
+      this.onStateChange(this.open);
+      fireCustomEvent('axa-state-change', { isOpen: !this.open }, this);
```

Comment:
This looks incorrect. First, the value of isOpen is not the same as the parameter passed to onStateChange, contrary to your docs. Second, you needlessly wrap an object around the event.detail value, contrary to your docs.

This seems correct in my view, taking your docs as spec:

fireCustomEvent('axa-state-change', this.open, this);